pkg/apis/autoscaling/annotation_validation.go

```diff
@@ -115,3 +115,32 @@ func validateMinMaxScale(annotations map[string]string) *apis.FieldError {
 	}
 	return errs
 }
+
+func validateMetric(annotations map[string]string) *apis.FieldError {
+	if metric, ok := annotations[MetricAnnotationKey]; ok {
+		var classValue string
```

Comment:
```suggestion
		classValue = KPA
```
and remove the else case.